sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

```diff
@@ -292,8 +297,11 @@ public void AddLink(string traceparent, string tracestate, IDictionary<string, s
             public Activity? Start()
             {
                 _currentActivity = StartActivitySourceActivity();
-
-                if (_currentActivity == null)
+                if (_currentActivity != null)
+                {
+                    _currentActivity.AddTag("az.schema_url", "https://opentelemetry.io/schemas/1.17.0");
```

Comment:
nit: my crystal ball tells me we'll need to support another version soon, so it'd be nice to move it to some constant and update all at once along with tests :)